environment.yml

```diff
@@ -5,7 +5,7 @@ dependencies:
   # ===========================
   # Dependencies from setup.cfg
   # ===========================
-  - python=3.11
+  - python >= 3.12.0
```

Comment:
Why is this necessary?

Reply:
This was just a maintenance dependecy bump. Since readthedocs built successfully, I thought why not updating to the latest python version. Did you have any reason to stay one version behind?

Reply:
It's okay to update the default Python version used in the build environment, I was just wondering if it was a constraint for `scipy`

Reply:
Nope no reason. I was just passing through the yaml file and I saw this pinning that could have been updated.